test/mystuff/main.go

```diff
@@ -0,0 +1,100 @@
+package main
```

Comment:
Can you rename this folder from mystuff to something else if we intend to keep this file ?

Reply:
whoops shouldn't have added this